circle.yml

```diff
@@ -25,7 +25,7 @@ test:
           - server/**/*.jsx
     - MOCHA_FILE=./test-results-client.xml npm run test-client -- -R mocha-junit-reporter -t $CIRCLE_NODE_TOTAL -i $CIRCLE_NODE_INDEX:
         parallel: true
-    - MOCHA_FILE=./test-results-client.xml npm run test-desktop -- -R mocha-junit-reporter -t $CIRCLE_NODE_TOTAL -i $CIRCLE_NODE_INDEX:
+    - npm run test-desktop:
```

Comment:
It takes almost 9 minutes to finish desktop test:

![screen shot 2017-05-02 at 14 52 04](https://cloud.githubusercontent.com/assets/699132/25618355/0128bd28-2f47-11e7-99a9-
90b7a38e2544.png)

It is also executed on all 4 containers with the same tests, because `parallel` flag is enabled. More info about `parallel` can be found here: https://circleci.com/docs/1.0/parallel-manual-setup/.